addons/point_of_sale/static/src/customer_display/customer_display.xml

```diff
@@ -10,14 +10,21 @@
                 <div class="position-absolute bottom-0 mb-4 d-none d-lg-flex align-items-center ps-3 pe-2 py-1 rounded-3 text-bg-dark small">Powered by <OdooLogo style="'width: 3rem;'" monochrome="true"/></div>
             </div>
             <div class="o_customer_display_main d-flex flex-column flex-grow-1 overflow-auto">
-                <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" class="'gap-0 p-0 pb-3 bg-view'" style="'scroll-snap-type: y mandatory;'">
-                    <Orderline line="scope.line" class="{
-                            'o_customer_display_orderline mx-2 bg-white fs-3 rounded-0': true,
-                        }"
-                    />
-                </OrderWidget>
-                <div t-else="" class="d-flex flex-column align-items-center justify-content-center h-100 w-100 text-muted">
-                    <h3 class="w-75 mt-2 text-center">Thank you</h3>
+                <t t-if="order.lines?.length and !order.finalized">
+                    <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" class="'gap-0 p-0 pb-3 bg-view'" style="'scroll-snap-type: y mandatory;'">
+                        <Orderline line="scope.line" class="{
+                                'o_customer_display_orderline mx-2 bg-white fs-3 rounded-0': true,
+                            }"
+                        />
+                    </OrderWidget>
+                </t>
+                <t t-elif="!order.lines?.length and !order.finalized">
+                    <div class="d-flex flex-column align-items-center justify-content-center h-100 w-100">
+                        <h1 class="display-2 w-75 mt-2 fw-bold text-center text-muted">Welcome.</h1>
+                    </div>
+                </t>
```

Comment:
Remove this change here. Look for the `emptyCartText` and change its return value to `_t("Welcome.")`.

Reply:
Hello @caburj,
OrderWidget is a generic component. If I change the emptyCartText for it, it will also be reflected in other places wherever it is being used. Also, there is one "cart" icon in the generic component, which we don't need in the customer display as per this task specs. So, what should I do ?